github/ci/prow-deploy/files/jobs/kubevirt/hostpath-provisioner/hostpath-provisioner-presubmits.yaml

```diff
@@ -21,7 +21,7 @@ presubmits:
       nodeSelector:
         type: bare-metal-external
       containers:
-        - image: gcr.io/k8s-testimages/bootstrap:v20190516-c6832d9
+        - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
```

Comment:
Maybe we should test this before, there are some differences between these images, for instance `gcr.io/k8s-testimages/bootstrap` is based on debian and `quay.io/kubevirtci/bootstrap` is based on fedora.

Reply:
I just wanted to be consistent. this repo already has scripts that we can alter to fix any issues we find.